ckb: init at 0.2.6

kierdavis opened the pull request:
ckb is a driver for Corsair keyboards/mice. It also contains a graphical tool for configuring their LED backlight settings.

The driver is implemented as a userland daemon. A NixOS module is included that runs this as a systemd service.

###### Motivation for this change


###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

pSub commented on pkgs/tools/misc/ckb/default.nix:
```diff
@@ -0,0 +1,40 @@
+    sha256 = "04h50qdzsbi77mj62jghr52i35vxvmhnvsb7pdfdq95ryry8bnwm";
+  };
+
+  buildInputs = [
```
pkgconfig and qmakeHook can be nativeBuildInputs.

kierdavis replied:
Thanks for the heads up. For future reference, what's the difference between `buildInputs` and `nativeBuildInputs`?

Mic92 replied:
@kierdavis I can only quote this one https://github.com/NixOS/nixpkgs/issues/19370

kierdavis committed:
ckb: init at 0.2.6

ckb is a driver for Corsair keyboards/mice. It also contains a graphical tool for configuring their LED backlight settings.

The driver is implemented as a userland daemon. A NixOS module is included that runs this as a systemd service.
